policyengine_us/variables/gov/states/id/tax/income/deductions/retirement_benefits/id_retirement_benefits_deduction.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class id_retirement_benefits_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho retirement benefits deduction"
+    unit = USD
+    documentation = "https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3022a/"
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.id.tax.income.deductions.retirement_benefits
+        filing_status = tax_unit("filing_status", period)
+
+        eligible_person = person(
+            "id_retirement_benefits_deduction_eligible_person", period
+        )
+        # Base retirement benefits deduction amount
+        base_amt = p.amount[filing_status]
```

Comment:
```suggestion
        max_amt = p.amount[filing_status]
```

Reply:
this is the max amount, we need to cap it also at the amount in Part Two:
https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=31